### ISSUE TITLE
Faiss Powered Multi-Label Classification

### ISSUE THREAD

**Issue by GeorgePearse** [scikit-learn-contrib/DESlib#257]
Do any of the implementations of KNN within this repo extend to the multi-label case? That is to say, usable in the context of multi-object detection from neural network embeddings or similar.

I was particularly hopeful that KNNE might work? But it's not obvious from the relevant literature.

Thanks a lot.

**Comment by jayahm** [comment]
Hi @GeorgePearse 

Sorry for the interruption.

Did you manage to perform DS on multi-label datasets?

I think the discussion is also related to my question [here](https://github.com/scikit-learn-contrib/DESlib/issues/267).

The issue is with y_val needs to be a 1d array, while target values/labels for multi-label datasets are more than 1 dimension.
